### PR TITLE
First Deliverable golang bootcamp

### PR DESCRIPTION
**Pull request by JulioCesarParra**
please review my code. thanks

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
```
Comment (alvarocueva) [inline main.go:3]: As a best practice, the imports should be separated as follows:
 ```go
 import(
     Go core
     
     Project local
     
     Third party
 )
 ```

File: main.go [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+	"fmt"
+	"net/http"
+	entity "academy-go-q42021/pkg/entity"
+    repo "academy-go-q42021/pkg/repo"
+	server "academy-go-q42021/pkg/server"
+	"os"
+	"log"
+)
+
+type Items []entity.Item
```
Comment (alvarocueva) [inline main.go:15]: By convention, every exported stuff (function, variable, constant, etc) should have a description
```go
// Items - Description here....
type Items []entity.Item
```

File: main.go [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+
```
Comment (alvarocueva) [inline main.go:17]: Remove this extra empty line

File: pkg/entity/entity.go [hunk 4]
```diff
@@ -0,0 +1,11 @@
+package entity
+
+type Item struct {
+	Id   int 	`json:"id"`
```
Comment (alvarocueva) [inline pkg/entity/entity.go:4]: By convention, the acronyms should be in uppercase. `Id` -> `ID`

File: pkg/server/api.go [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+func (a *api) homePage(w http.ResponseWriter, r *http.Request) {
+	fmt.Fprintf(w,"Welcome to the HomePage!")
+	fmt.Println("Endpoint Hit: homePage")
+}
+
+func (a *api) fetchItems(w http.ResponseWriter, r *http.Request) {
+	fmt.Println("Endpoint Hit: fetchItems")
+	records, _ := a.repository.FetchItems()
```
Comment (alvarocueva) [inline pkg/server/api.go:44]: Handle the error

**Comment by alvarocueva** [comment]
Please change the PR's base, to avoid merging into **_Wizeline Academy_** repo, the merge should be to your same fork.
If you can't change it, don't worry, just decline the PR and do your push to your fork.
For the next PR's point the PR's merge to your fork.

**Comment by JulioCesarParra** [comment]
closing .  
This PR will be worked with the next PR link
https://github.com/JulioCesarParra/academy-go-q42021/pull/1

**Pull request closed without merging**